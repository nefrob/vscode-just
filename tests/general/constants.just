# Constants

true
false

HEX
HEXLOWER
HEXUPPER
