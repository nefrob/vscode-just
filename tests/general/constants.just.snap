># Constants
#^^^^^^^^^^^ source.just comment.line.number-sign.just
>
>true
#^^^^ source.just constant.language.boolean.just
>false
#^^^^^ source.just constant.language.boolean.just
>
>HEX
#^^^ source.just constant.language.hex.just
>HEXLOWER
#^^^^^^^^ source.just constant.language.hex.just
>HEXUPPER
#^^^^^^^^ source.just constant.language.hex.just
>